packages/editor/src/Sx/Space.tsx

```diff
@@ -67,64 +74,59 @@ export const Space = ({
     }
   }
 
-  const prefixName = name => tag ? `styles.${tag}.${key}${name}` : key + name
-  const label = dir => property === 'margin' ? 'Margin ' + dir : 'Padding ' + dir
+  const prefixName = name => (tag ? `styles.${tag}.${key}${name}` : key + name)
+  const label = dir =>
+    property === 'margin' ? 'Margin ' + dir : 'Padding ' + dir
 
   return (
     <div
       sx={{
         display: 'grid',
-        gridGap: 2,
+        gap: 2,
         gridTemplateColumns: 'repeat(3, 1fr)',
         alignItems: 'center',
       }}>
-      <Field
+      <Field<'input'>
```

Comment:
Oh, is it still needed after the update to @theme-ui/components?

Reply:
Fixed in 698838e0.